app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/profile/ProfileViewModel.kt

```diff
@@ -16,7 +15,7 @@ class ProfileViewModel : BaseViewModel() {
     private val repo: BitbucketRepository by inject()
 
     // UI
-    val avatarUrl: Flow<String> = repo.user.map { it?.convertToUser()?.avatarUrl.orEmpty() }
+    val avatarUrl: Flow<String> = repo.user.map { it?.avatarUrl.orEmpty() }.groundState("")
```

Comment:
@BottleRocket-Colin Since you're converting to a `StateFlow` via `groundState`, did you also want to update the signature of the property to take advantage of that more concrete type? If not, what exactly are you gaining by adding `groundState` here?

Reply:
Since I'm only using the interface provided/defined by Flow it's ok to leave this typed as a Flow.  The instantiated object is still a StateFlow and I'm getting the benefit of the internal statefulness.  Although, for documentation purposes it would be a good idea to change it.  So I will :)